code/modules/mob/living/living_defines.dm

```diff
@@ -211,3 +211,7 @@
 
 	/// Living mob's mood datum
 	var/datum/mood/mob_mood
+
+	// Multiple imaginary friends!
+	/// Contains the owner and all imaginary friend mobs if they exist, otherwise null
+	var/list/imaginary_group = null
```

Comment:
I think I can definitely put it onto the brain trauma level. I made sure it wouldn't impact performance by only initialising it when someone gets the trauma at least. I'll try to move it

Reply:
I thought about it a bit more, I think this may be the most elegant way of doing it. Putting it on the disease itself would mean either ensuring there's only one person or managing multiple lists which could be all over the place. We could also check for existing diseases and use that list. If you can think of a more elegant way of doing it please let me know.